src/mibase.ts

```diff
@@ -531,6 +531,7 @@ export class MI2DebugSession extends DebugSession {
 				this.sendErrorResponse(response, 1, `Could not expand variable: ${err}`);
 			}
 		} else if (typeof id == "string") {
+			id.replace(/^static /, "");
```

Comment:
I wasn't aware that this changes the content of the `id` object and that this can be done to a const

Reply:
> I wasn't aware that this changes the content of the `id` object and that this can be done to a const

The code is useless; const variables do not have a const modifier.